tests/validation_suite/automation_framework/work_order_submit/work_order_submit_params.py

```diff
@@ -0,0 +1,643 @@
+        Parameters:
+            - data is each item in inData or outData part of workorder request
+              as per TCF API 6.1.7 Work Order Data Formats
+            - encryption_key is the key used to encrypt the data
+            - iv is an initialization vector if required by the data encryption algorithm.
+              The default is all zeros.iv must be a unique random number for every
+              encryption operation.
+        """
+        logger.debug("encrypted_session_key: %s", encryption_key)
+        if iv is not None:
+            encrypted_data = crypto.SKENC_EncryptMessage(encryption_key, iv, data)
+        else:
+            encrypted_data = crypto.SKENC_EncryptMessage(encryption_key, 0,  data)
+        return encrypted_data
+
+    def get_verifying_key(self):
+        if "verifyingKey" in self.params_obj:
+            return self.params_obj["verifyingKey"]
+        else :
+            return None
+
+    def get_params(self):
+        params_copy = self.params_obj.copy()
+        if "inData" in params_copy:
+            params_copy.pop("inData")
+        if "outData" in params_copy:
+            params_copy.pop("outData")
+        return params_copy
+
+    def get_in_data(self):
+        if "inData" in self.params_obj:
+            return self.params_obj["inData"]
+        else :
+            return None
+
+    def get_out_data(self):
+        if "outData" in self.params_obj:
+            return self.params_obj["outData"]
+        else :
+            return None
+
+    def get_requester_nonce(self):
+        return self.params_obj["requesterNonce"]
+
+    def get_worker_id(self):
+        if "workerId" in self.params_obj:
+            return self.params_obj["workerId"]
+        else :
+            return None
+
+    def get_workload_id(self):
+        if "workloadId" in self.params_obj:
+            return self.params_obj["workloadId"]
+        else :
+            return None
+
+    def get_requester_id(self):
+        if "requesterId" in self.params_obj:
+            return self.params_obj["requesterId"]
+        else :
+            return None
+
+    def get_session_key_iv(self):
+        return self.params_obj["sessionKeyIv"]
+
+    def get_work_order_id(self):
+        if "workOrderId" in self.params_obj:
+            return self.params_obj["workOrderId"]
+        else :
+            return None
+
+    def get_encrypted_session_key(self):
+        return self.params_obj["encryptedSessionKey"]
+
+    def get_encrypted_request_hash(self):
+        if "encryptedRequestHash" in self.params_obj:
+            return self.params_obj["encryptedRequestHash"]
+        else :
+            return None
+
+    def get_requester_signature(self):
+        if "requesterSignature" in self.params_obj:
+            return self.params_obj["requesterSignature"]
+        else :
+            return None
+
+    def verify_signature(self, input_json, verification_key):
+        """
```

Comment:
This is same as utility in source code. Can you directly take the utility function instead of duplicating this.